examples/controlnet/README_sdxl.md

```diff
@@ -66,6 +66,7 @@ export OUTPUT_DIR="path to save model"
 
 accelerate launch train_controlnet_sdxl.py \
  --pretrained_model_name_or_path=$MODEL_DIR \
+ --pretrained_vae_model_name_or_path "madebyollin/sdxl-vae-fp16-fix" \ # Vae fix is required to converge with dataset "fusing/fill50k"
```

Comment:
```suggestion
 --pretrained_vae_model_name_or_path="madebyollin/sdxl-vae-fp16-fix" \ # Vae fix is required to converge with dataset "fusing/fill50k"
```

* Could we maybe assign the VAE path as an env var and then use it in the command following the rest of the command examples?
* "# Vae fix is required to converge with dataset "fusing/fill50k"" -- could we maybe remove this from the command and instead make this note just below the command?